pom.xml

```diff
@@ -252,7 +252,10 @@
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.5.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
```

Comment:
Check if scope is test

Reply:
Not only required for tests. It's being used also.